_posts/2019-01-31-the-build-system.md

```diff
@@ -0,0 +1,202 @@
+---
+title: The Build System
+layout: post
+categories: [blog]
+tags: [build system,architecture]
+author: varl
+---
+
+The last year has seen some significant changes to how the build system
+operates. There are a few moving pieces, so let's get started!
+
+![](/assets/build_arch/build_arch.png)
```

Comment:
What about this first diagram being more high level. As in, don't show all the details inside the box, but instead have the main focus on the various services that are part of the system.  Maybe just opaque boxes with the service name, and the most basic input and output of each box

Just a thought.

Reply:
Good input, thanks. The initial idea was to show the entire thing, then break it up into pieces (using the exact same boxes), so at a second glance the reader could recognize each piece in the greater whole. Might work better in my head though. I'll simplify the conceptual diagram. :ok_man: